doc/modules/changes/20210128_bobmyhill

```diff
@@ -0,0 +1,16 @@
+Changed: The reference temperature for the multicomponent incompressible
+equation of state is now set to either the adiabatic surface temperature
+(if adiabatic heating is turned off), or the
+temperature along the reference adiabat at the local static pressure
+(when adiabatic heating is turned on).
+If users have previously used this model with adiabatic heating turned off
+and a user-defined reference temperature that is not equal to the
+adiabatic surface temperature, they should change the reference densities
+in their model according to the expression
+new_densities[i] = old_densities[i] * (1 - thermal_expansivities[i] *
+(adiabatic_surface_temperature - old_reference_temperature)). The results
```

Comment:
This is not done in the updated parameter files, but from the discussion it looks like there may be a reason for this? My first reaction is that many of the changed cookbooks, benchmarks, and tests should have the reference densities updated to get as close as possible to the original density profiles, but I admit I don't fully understand all of the related thermodynamic arguments.

Reply:
The reason I haven't done this is because a) I don't think any of the tests use real densities measured at the reference temperature and b) the test results would still be different.

I looked through the cookbooks and benchmarks, and it wasn't clear to me whether the authors cared about the exact density. @gassmoeller and @dannberg told me that a lot of users didn't understand what the parameter was, so I've opted for keeping simple round numbers. Happy to change this if the cookbooks/benchmarks require a specific density on the reference isentrope.

Reply:
I've also changed this description so that it no longer contains the equation. Users could calculate the density and thermal expansivity by extrapolation or direct measurement.

Reply:
I think we will definitely want to adjust the reference densities for the following cases:

- continental extension cookbook 
- Buiter et al., 2008, 2016 benchmarks
- Crameri et al. benchmark
- Slab detachment benchmark

All of these should be fairly straightforward changes to the reference density, except the continental extension benchmark will also require updates to the manual. It has been on my list to update that benchmark to make a number of other changes to this benchmark - shall I submit a separate PR for this in the next few days and then this PR can be rebased?

Reply:
Hi @naliboff,

I guess there are two questions here. 
* To which temperature was the reference density originally meant to correspond?
* How much do we care about new results being close (i.e. not identical) to the originals?

I've noticed that in a lot of prms, the adiabatic surface temperature is not changed from the default value (0 K). I don't know whether this is intentional, but even for crustal modelling on Earth that seems like an odd choice. So if these models are to be changed, maybe this parameter could also be modified?

Reply:
Sorry, just saw this discussion after my comment below. Yes, for many models the fact that adiabatic surface temperature was not set highlights the problem. Feel free to modify that parameter to the value of the reference temperature from the material model. That should fix a lot of the changes.